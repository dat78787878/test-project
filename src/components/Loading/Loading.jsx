import React from 'react';
import Spinner from 'react-bootstrap/Spinner';
const Loading = () => {
  return (
    <div className="loading">
      <Spinner animation="border" variant="success" data-testid="loading" />
    </div>
  );
};

export default Loading;
