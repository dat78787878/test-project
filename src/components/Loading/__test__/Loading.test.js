import React from 'react';
import { render } from '@testing-library/react';
import Loading from '../Loading';
window.React = React;

describe('Loading component', () => {
  it('render component success', () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
