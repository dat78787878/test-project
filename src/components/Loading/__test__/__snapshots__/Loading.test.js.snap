// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component render component success 1`] = `
<div>
  <div
    class="loading"
  >
    <div
      class="spinner-border text-success"
      data-testid="loading"
    />
  </div>
</div>
`;
